### ISSUE TITLE
Implement EthereumSignMessage

### ISSUE THREAD

**Issue by LogvinovLeon** [trezor/connect#66]
It's already done in the newest firmware: https://github.com/trezor/trezor-mcu/pull/189

**Comment by prusnak** [comment]
Would you like to try to implement it in connect? The API is very similar to Bitcoin Sign/Verify Message.

**Comment by prusnak** [comment]
See here: https://github.com/trezor/trezor-common/blob/master/protob/messages.proto

**Comment by LogvinovLeon** [comment]
I've already implemented it for myself locally, but it's hard to test it, cause it also meeds to be implemented in popup.js, which is fetched from your s3 bucket.
I created s3 bucket for myself and also implemented it in popup.js and redirected connect to this bucket, but I'm still getting `Could not establish connection` errors.
I guess it might have smth to do with me using a different origin and  you checking the signatures from config_signed.bin.

TL;DR I'll start a PR in this repo, but i'm unable to test it.

**Comment by prusnak** [comment]
No worries about testing, we will test it.

**Comment by prusnak** [comment]
Let's continue the discussion in #66. Closing this

**Comment by lyddream** [comment]
Is ethereumSignMessage the same as ethereum eth_sign?
does ethereumSignMessage method add ethereum prefix(\x19Ethereum Signed Message:\n) during sign progress?

**Comment by lyddream** [comment]
const hash = hashPersonalMessage(toBuffer('test')).
const sig = ecsign(hash,privateKey)

I  use the above method to sign a message ,but I can' t get the same result from ethereumSignMessage?

**Comment by prusnak** [comment]
Ethereum Sign Message is broken across implementation. There are many incompatible standards. We will implement one which is universally adopted. Until then let's consider this as unimplemented. 

Closing as duplicate of https://github.com/trezor/trezor-core/issues/122
